spec/unit/tester/test_asserter.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+module Webspicy
+  class Tester
+    describe Asserter do
+      let(:asserter) { Asserter.new(target) }
+
+      describe '#size' do
```

Comment:
@blambeau The way I usually structure the describes and contexts is slightly different from yours. If it seems jarring to have different styles in the same project, I'll change this to conform to existing specs (for now :smiley: )

Reply:
I don't care about the style of the tests. I even like seeing & learning other ways. No need to change anything.